src/components/contract/ContractResult.vue

```diff
@@ -118,6 +118,9 @@
 
     <ContractResultLogs :logs="contractResult?.logs" :block-number="blockNumber" :transaction-hash="transactionHash"/>
 
+    <!--Workaround to ensure margin bottom-->
+    <div />
+
```

Comment:
@Sheng-Long  you may rebase your branch onto main. You will get a small fix which solves such margin discrepancies at the end of the pages.
You can get rid of that workaround `<div>`